class/metallb.yml

```diff
@@ -1,5 +1,11 @@
 parameters:
   kapitan:
+    dependencies:
+      - type: git
```

Comment:
Is there a reason you're using a `git` dependency here, instead of [Helm](https://kapitan.dev/external_dependencies/#helm-type)?

Reply:
Yes, Bitnami has decided to have one big repo. Couldn't find any other source for the Helm Chart and those Repo does not have any tags or better references. While MetalLB is taking over the Bitnami Helm Chart into their own Repo https://github.com/metallb/metallb/issues/653, this is an intermediate "issue".

Reply:
You can just use the proper chart. No need to clone it via chart:
```
  dependencies:
    - type: helm
      output_path: dependencies/metallb/helmcharts/
      source: https://charts.bitnami.com/bitnami
      version: ${metallb:charts:metallb}
      chart_name: metallb
```

And then you can set `metallb:charts:metallb` to the actual chart version: `0.1.23` https://github.com/bitnami/charts/blob/master/bitnami/metallb/Chart.yaml#L4